MumblePAHelper.pro

```diff
@@ -19,4 +18,14 @@ HEADERS  += mumblepahelper.h \
 
 FORMS    += mumblepahelper.ui
 
-CONFIG += c++11
\ No newline at end of file
+win32:RC_ICONS += resources/MumblePAHelper.ico
+
+CONFIG += c++11
+
+Release: DESTDIR = $$_PRO_FILE_PWD_/release
```

Comment:
(You'd probably use `DESTDIR = $$_PRO_FILE_PWD_/release`, and s/release/debug, etc.)